src/callbacks_step/summary.jl

```diff
@@ -20,9 +20,14 @@ function SummaryCallback(reset_threads = true)
         initialize_summary_callback(cb, u, t, integrator;
                                     reset_threads)
     end
+    # At the end of the simulation, the timer is printed
+    function finalize(cb, u, t, integrator)
+        finalize_summary_callback(cb, u, t, integrator)
+    end
     DiscreteCallback(summary_callback, summary_callback,
                      save_positions = (false, false),
-                     initialize = initialize)
+                     initialize = initialize,
+                     finalize = finalize)
```

Comment:
```suggestion
                     finalize = finalize_summary_callback)
```

Reply:
Then should we do the same for `initialize_summary_callback`?

Reply:
Ah, I see.